graph/RM6175_Records_Information_Management_Service.cypher

```diff
@@ -0,0 +1,60 @@
+(ansCombinedDigital:Answer {uuid: '5be4bf0e-73e8-455f-b560-e66734eaf278', text: 'Combined digital workflow, cloud based hosting and records information management services', hint: 'Provides cloud hosting and short-term technical services to help you reduce your physical records and move to a digital storage solution. You can also access offsite management services for physical records'}),
+(ansSomethingElse:Answer {uuid: 'd983b430-92f5-42f2-b778-3a5bab9cf4a2', text: 'Something else', hint: 'What I need is not listed here'}),
+
+// Tree Structure
+(jrnyRecordInfo:Journey {uuid: 'd39ea9e1-b33c-4ce7-9ec2-6448d8277684', name: 'Records Information Management Service'}),
+(jrnyRecordInfo)-[:FIRST_QUESTION]->(qiFirstQuestion:QuestionInstance:Outcome {uuid: '8b608d94-7b9d-4012-b2a3-6217873a4198'})-[:DEFINED_BY]->(qstnService),
+
+    (ansGrpRecordInfo:AnswerGroup {name: 'ansGrpRecordInfo'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpRecordInfo),
+    (ansGrpRecordInfo)-[:HAS_ANSWER {order: 1}]->(ansRecordInfo),
+    (ansGrpRecordInfo)-[:HAS_OUTCOME]->(:Agreement:Outcome {number: 'RM6175'})-[:HAS_LOT]->(:Lot {number: '1', url: '', type: 'CAT', scale: true}),
+
+    (ansGrpDigitalCloud:AnswerGroup {name: 'ansGrpDigitalCloud'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpDigitalCloud),
+    (ansGrpDigitalCloud)-[:HAS_ANSWER {order: 2}]->(ansDigitalCloud),
+    (ansGrpDigitalCloud)-[:HAS_OUTCOME]->(:Agreement:Outcome {number: 'RM6175'})-[:HAS_LOT]->(:Lot {number: '2', url: '', type: 'CAT', scale: true}),
+
+    (ansGrpNHSRecord:AnswerGroup {name: 'ansGrpNHSRecord'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpNHSRecord),
+    (ansGrpNHSRecord)-[:HAS_ANSWER {order: 3}]->(ansNHSRecord),
+    (ansGrpNHSRecord)-[:HAS_OUTCOME]->(:Agreement:Outcome {number: 'RM6175'})-[:HAS_LOT]->(:Lot {number: '3', url: '', type: 'CAT', scale: true}),
+
+    (ansGrpSpecialistRecord:AnswerGroup {name: 'ansGrpSpecialistRecord'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpSpecialistRecord),
+    (ansGrpSpecialistRecord)-[:HAS_ANSWER {order: 4}]->(ansSpecialistRecord),
+    (ansGrpSpecialistRecord)-[:HAS_OUTCOME]->(:Agreement:Outcome {number: 'RM6175'})-[:HAS_LOT]->(:Lot {number: '4', url: '', type: 'CAT', scale: true}),
+
+    (ansGrpCombinedDigital:AnswerGroup {name: 'ansGrpCombinedDigital'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpCombinedDigital),
+    (ansGrpCombinedDigital)-[:HAS_ANSWER {order: 5}]->(ansCombinedDigital),
+    (ansGrpCombinedDigital)-[:HAS_OUTCOME]->(:Agreement:Outcome {number: 'RM6175'})-[:HAS_LOT]->(:Lot {number: '5', url: '', type: 'CAT', scale: true}),
+
+    (ansGrpSomethingElse:AnswerGroup {name: 'ansGrpSomethingElse'}),
+    (qiFirstQuestion)-[:HAS_ANSWER_GROUP]->(ansGrpSomethingElse),
+    (ansGrpSomethingElse)-[:HAS_ANSWER {order: 6}]->(ansSomethingElse),
+    (ansGrpSomethingElse)-[:HAS_OUTCOME]->(resultCCSEscapePage)
```

Comment:
needs to add `(resultCCSEscapePage:Support {uuid: 'ccb5beb6-75b5-11ea-bc55-0242ac130003'}),` in your match section